README.md

```diff
@@ -119,7 +119,7 @@ To import pod you should add string:
 - For `Swift` projects:
 
 
-        import CoreActionSheetPicker
+        import ActionSheetPicker_3_0

```

Comment:
Are you sure in that?
Seems you have to write CoreActionSheetPicker for import. Did you try it?